content/en/docs/install-setup/setup/install/_index.md

```diff
@@ -1,11 +1,15 @@
 ---
-title: Install jx
-linktitle: Install jx
+title: Install the jx binary
+linktitle: Install the jx binary
 description: How to install the jx binary on your machine?
-weight: 5
+weight: 1
```

Comment:
For what it's worth, there is an argument for not incrementing by one. If you do, you will need to reshuffle all weights each time you want to put something new in. If one increments by say 10, there is some room for adding pages w/o having to change weights all the time.

Reply:
+1 on specifying at least 10 for **weight** on the font matter to leave room for other pages.

Reply:
ack. I've changed the weighting to increments of 10. Will be in next push.